src/client/src/components/pages/courses/addCourses/index.jsx

```diff
@@ -25,17 +27,60 @@ export default class index extends Component {
     this.setState(fields);
   };
 
+  addCourse = obj => {
+    swal({
+      type: "warning",
+      html: "Are you sure that you want to add this course ?",
+      showCancelButton: true,
+      focusConfirm: false,
+      confirmButtonText: '<i className="fa fa-thumbs-up"></i> Yes',
+      confirmButtonAriaLabel: "Thumbs up",
+      cancelButtonText: '<i className="fa fa-thumbs-down"></i> No ',
+      cancelButtonAriaLabel: "Thumbs down"
+    }).then(confirm => {
+      if (confirm.value) {
+        axios('/api/v2/courses', {
+          method: 'POST',
+          data: {
+            courseData: obj
+          }
+        }).then(result => {
+          if (result.data.error) {
+            swal({
+              title: "",
+              type: "warning",
+              html: result.data.error,
+              confirmButtonText: "Ok"
+            }).then(confirm => {
+              if (confirm.value) {
```

Comment:
You might not need this if-statement, it's probably safe to assume that you should always do the redirect to courses/view after this alert is finished.